app/imdb_tools.py

```diff
@@ -38,10 +38,10 @@ def imdb_get_movies(config_path, plex, data):
                            "//a/img//@data-tconst")
     if "/search/" in imdb_url:
         results = re.search('<span>\\d+-\\d+ of \\d+ titles.</span>', str(r.content))
-        total = 100 if results is None else re.findall('(\\d+)', results.group(0))[2]
+        total = 100 if results is None else re.findall('(\\d+)', results.group(0).replace(',', ''))[2]
```

Comment:
In the future, should we just set the size as a parameter and add it to the URL on the user's behalf?

Reply:
i mean this solution should work and then if the list changes its total the user would have to constantly update the total